plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -138,6 +139,15 @@ func findNodesThatFit(pod *api.Pod, podLister algorithm.PodLister, predicateFunc
 			filtered = append(filtered, node)
 		}
 	}
+	if len(filtered) > 0 && len(extenders) != 0 {
+		for _, extender := range extenders {
+			filteredList, err := extender.Filter(pod, &api.NodeList{Items: filtered})
```

Comment:
I would like to have a clean split of responsibilities between k8s scheduler and the extender. k8s will do the cpu/mem/label/... based scheduling. extender is limited to solving use cases beyond what k8s provides, like storage and network aware scheduling. I dont want the extender to be very rich, then it gravitates towards being another scheduler on its own.